src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -29,6 +29,8 @@ class SearchHelpSignIn extends Component {
     this.showHomepageCreateAccountBlock();
   }
 
+  signInVis = window.location.pathname === '/sign-in/' ? 'hidden' : 'visibile';
+
```

Comment:
No. Let's use CSS in our SignInApp to update styles

Reply:
updated